### ISSUE TITLE
Lab 09A - Reference to Fields pane

### ISSUE THREAD

**Issue by VitalyMCT** [MicrosoftLearning/DA-100-Analyzing-Data-with-Power-BI#25]
> In Power BI Desktop, go to the My Performance page, and in the **Fields** pane, remove the Salesperson filter card.

The intended pane is Filters rather than Fields. The screenshot is correct, so just the description should be adjusted.

**Comment by FraSkills3362** [comment]
I agree,
Lab 09A
Exercise 5
Task 1
Step 2
Change:
In Power BI Desktop, go to the My Performance page, and in the **Fields** pane, remove the Salesperson filter card.
To:
In Power BI Desktop, go to the My Performance page, and in the **Filters** pane, remove the Salesperson filter card.

**Comment by luconsta** [comment]
I agree with @FraSkills3362 except the line to change (at least at date I'm writing) is in **Lab 09A / Exercise 5/Finish Up** (not Task1)**/Step 3** (not Step2).
Even the picture below text it shows the **Filters** pane not the **_Fields_** pane.
